core/src/test/java/com/exadel/aem/toolkit/api/annotations/injectors/RequestSelectorsInjectorIntegrationTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exadel.aem.toolkit.api.annotations.injectors;
+
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.exadel.aem.toolkit.api.annotations.injectors.models.TestModelSelectors;
+
+import io.wcm.testing.mock.aem.junit.AemContext;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+
+public class RequestSelectorsInjectorIntegrationTest {
```

Comment:
...Integration ?
For me, it looks like a typical unit test)

Reply:
I thought if we are using Context, and testing interaction between model and service this is an integration test. I'll rename to test.